src/custom/components/SearchModal/ManageLists/index.tsx

```diff
@@ -4,12 +4,26 @@ import { DEFAULT_NETWORK_FOR_LISTS, UNSUPPORTED_LIST_URLS } from 'constants/list
 import { useActiveWeb3React } from '@src/hooks'
 import { CurrencyModalView } from '@src/components/SearchModal/CurrencySearchModal'
 import { TokenList } from '@uniswap/token-lists'
+import { acceptListUpdate, removeList, disableList, enableList } from 'state/lists/actions'
+
+export interface ListRowProps {
+  acceptListUpdate: (url: string) => ReturnType<typeof acceptListUpdate>
+  removeList: (url: string) => ReturnType<typeof removeList>
+  disableList: (url: string) => ReturnType<typeof disableList>
+  enableList: (url: string) => ReturnType<typeof enableList>
+}
 
 export const ManageLists = (props: {
   setModalView: (view: CurrencyModalView) => void
   setImportList: (list: TokenList) => void
   setListUrl: (url: string) => void
 }) => {
   const { chainId = DEFAULT_NETWORK_FOR_LISTS } = useActiveWeb3React()
-  return <ManageListsMod {...props} unsupportedListUrls={UNSUPPORTED_LIST_URLS[chainId]} />
+  const listRowProps = {
```

Comment:
to minimise changes necessary inside the old comp, we build the `lists/actions` functions here to retain the same shape as previously but pass in `chainId` as sort of a wrapped function